webpack/templates/siteCard.handlebars

```diff
@@ -151,9 +151,11 @@
                 {{#if lastVerified}}
                     <div class="text-xs">{{t "verified_at" lastVerified=lastVerified}}</div>
                 {{/if}}
-                {{t "correction" }}
-                <a href="https://airtable.com/shrdt7TsVi76kTzYt?prefill_Location+ID={{id}}" target="_blank">{{t "correction_link"}}</a>
             </div>
         </div>
+        <div class="flex flex-row flex-wrap justify-between items-end px-4 py-2 text-sm">
+            <p>{{t "card_out_of_date"}}</p>
+            <p><a href="https://vaccines.gov">{{t "shutdown_forward_no_link"}}</a></p>
```

Comment:
```suggestion
            <p><a href="https://vaccines.gov">{{{t "shutdown_forward_no_link"}}}</a></p>
```

Should let the `&rarr;` turn into `→` properly

I'd also suggest putting this outside of the `<details></details>` element, so it shows up even if the user doesn't expand the information.

Reply:
Should the details just be removed entirely then?

Reply:
resolved by #af5f6d5